src/tests/test_parse.py

```diff
@@ -424,3 +425,52 @@ def test_connections_file_get_default_connection_url(tmp_empty, content, expecte
 
     cf = ConnectionsFile(path_to_file="conns.ini")
     assert cf.get_default_connection_url() == expected
+
+
+@pytest.mark.parametrize(
```

Comment:
add a unit test using duckdb (`duckdb.connect()`) and querying using the slicing notation, ensure that the output matches what you'd see if you passed this directly to duckdb (as opposed to using jupysql)

Reply:
@edublancas just to clarify, the test should connect to duckdb through `duckdb.connect()` and run a query like `conn.execute("select 'hello'[:2]").fetchall()`. then it should run the same query through jupysql  like `%%sql
select 'hello'[:2]` and compare the results?

Reply:
the test should be: run something via jupysql. e.g., `%%sql select 'hello'[:2]` and then check that the output matches what you'd expected from duckdb.

you can hardcode the output from duckdb in the test. e.g. assuming you run: `'hello'[:2]`, you can check `assert output == "he"`

Reply:
[Added test here](https://github.com/ploomber/jupysql/pull/910/files#diff-edd543175e806ac6654196270596c90240cc3a3cfdcba20d35d997207e4555f0R431-R458)